### ISSUE TITLE
error in CreateThumbnailModule.java IOException is never thrown

### ISSUE THREAD

**Issue by anirudh-antino** [souvik-ghosh/react-native-create-thumbnail#76]
```
node_modules/react-native-create-thumbnail/android/src/main/java/com/createthumbnail/CreateThumbnailModule.java:194: error: exception IOException is never thrown in body of corresponding try statement
        } catch(IOException e) {
          ^
```
```
 try {
            retriever.release();
        } catch(IOException e) { <------- here
            e.printStackTrace();
        }
```

**Comment by ZamirAmiri** [comment]
There is a catch already in the parent function just a ```throws IOException``` would suffice.

**Comment by fukemy** [comment]
I install v0.16.3 but still got this problem

**Comment by john-vuong** [comment]
This is the same [issue as react-native-cameraroll](https://github.com/react-native-cameraroll/react-native-cameraroll/issues/402). 

Set the compile/target SDK to version 33 (Android 13) to recreate the error.

The [fix](https://github.com/react-native-cameraroll/react-native-cameraroll/pull/419/) is simple.

Thanks!

**Comment by souvik-ghosh** [comment]
> ```
> node_modules/react-native-create-thumbnail/android/src/main/java/com/createthumbnail/CreateThumbnailModule.java:194: error: exception IOException is never thrown in body of corresponding try statement
>         } catch(IOException e) {
>           ^
> ```
> 
> ```
>  try {
>             retriever.release();
>         } catch(IOException e) { <------- here
>             e.printStackTrace();
>         }
> ```

The problem with adding this code is that the library won't work on Android version < 13 anymore. So we planned to support android 13 from v2.0.0 onwards. Currently, it's available as a pre-release `v2.0.0-rc.2`.

**Comment by souvik-ghosh** [comment]
**UPDATE:** v2.0.0 has been released with the fix
